### ISSUE TITLE
Menu Link Attributes module update to 1.2 causes PHP warnings

### ISSUE THREAD

**Issue by sejancich** [forumone/gesso#444]
See: https://www.drupal.org/project/menu_link_attributes/issues/3199777#comment-14036454

**Comment by johnbburg** [comment]
I believe this is happening on this line.

https://github.com/forumone/gesso/blob/0d7330aebb6a279cf7f19149d839c3490b61de0e/source/_patterns/04-components/menu/menu.twig#L46

```
        {% set link_classes = link_classes|merge([item['original_link'].options.attributes.class]) %}
```
Should remove the redundant square brackets inside of merge.
```
        {% set link_classes = link_classes|merge(item['original_link'].options.attributes.class) %}
```

Edit: whoops, didn't see that this was already being worked on in a PR. Diregard.

**Comment by coreylafferty** [comment]
Fixed by #421
